dist/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export default function VeeValidatePlugin(opts: any): (baseReturns: any) => any;
+export function withField(el: any): any;
+export function mapElementsInSchema(schema: any[], fn: Function): any[];
```

Comment:
These are probably exported by mistake, I didn't realize that till now.

Could we remove those `export` statements from the main `index.js` file? They shouldn't be used and there shouldn't be any named export when we export the plugin function by default.

Reply:
Hi @logaretm,

I removed export of these two functions and build again.
Would you please take a look and see if this is the desired result?